src/lib/stores/bookmarks.svelte.ts

```
@@ -0,0 +1,12 @@
+import { writable, type Writable } from 'svelte/store';
+
+let bookmarks: string[] = $state([]);
+export let storedBookmarks: Writable<string[]> = writable([]);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'storedBookmarks' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)